astroplan/plots/time_dependent.py

```diff
@@ -120,7 +118,8 @@ def plot_airmass(targets, observer, time, ax=None, style_kwargs=None,
     max_region : float
         If set, defines an interval between ``max_airmass`` and ``max_region``
         that will be shaded. Default is `None`.
-
+    moon_airmass : bool
```

Comment:
@cagostino it looks like you just forgot to include the `moon_airmass` parameter in the method declaration (lines 47-48).

I'd be curious about this PR. Can you make the fix above and post an example plot of what this looks like? Cheers!

Reply:
![3xmmhii0airmass](https://user-images.githubusercontent.com/5403925/49600199-a5d99480-f950-11e8-8078-2ecb4fdfd507.png)
I've added an example. Here red is the corresponding airmass for the moon. I should be able to make these suggested changes as well as soon as my semester ends.

If you'll notice, the brightness shading here also covers multiple days, which is the subject of a separate pull request (#354) and is not currently implemented in the master branch iirc.